api/android/api/src/main/jni/Android.mk

```diff
@@ -126,7 +132,7 @@ include $(LOCAL_PATH)/Android-gst-plugins.mk
 
 GSTREAMER_PLUGINS        := $(GST_REQUIRED_PLUGINS)
 GSTREAMER_EXTRA_DEPS     := $(GST_REQUIRED_DEPS) gio-2.0 gmodule-2.0 json-glib-1.0
-GSTREAMER_EXTRA_LIBS     := $(GST_REQUIRED_LIBS) -liconv
+GSTREAMER_EXTRA_LIBS     := $(GST_REQUIRED_LIBS) -liconv -lcairo
```

Comment:
Is this safe? (-lcairo)

Reply:
With build option, set '-lcairo' (build type all).
Also I updated '-lmediandk' when including amcsrc element.